### ISSUE TITLE
Dev - Favicon Should Be Updated To New Logo

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13901]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Run Expensify Locally
2. Open http://localhost:8081/

## Expected Result:
Favicon is not updated to new logo

## Actual Result:
Old favicon appears

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [ ] MacOS /Safari

**Version Number:** v1.2.45-0 Dev
**Reproducible in staging?:**
**Reproducible in production?:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
<img width="254" alt="Screenshot 2022-12-29 at 3 57 52 PM" src="https://user-images.githubusercontent.com/43996225/210027580-8126f027-f6b2-432b-9516-fdc5d8d4bf63.png">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @syedsaroshfarrukhdot 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1672311758561289

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by grgia** [comment]
I think this might be related to this: 
https://expensify.slack.com/archives/C049HHMV9SM/p1669750882214929

TLDR: The favicons have been updated, we decided not to automatically flush the cache when we updated the logos

**Comment by grgia** [comment]
<img width="1615" alt="image" src="https://user-images.githubusercontent.com/38015950/210100607-18cc57df-89b7-452a-bb3b-5a56365710ef.png">
Tested again in Safari, gonna close this one out!
